app/actions/ModActionPanel/BlobList.tsx

```diff
@@ -1,25 +1,36 @@
-import { ComponentProps } from 'react'
+import { ComponentProps, useState } from 'react'
 import { ToolsOzoneModerationDefs } from '@atproto/api'
 import { ShieldExclamationIcon } from '@heroicons/react/20/solid'
-import { formatBytes } from '@/lib/util'
+import { formatBytes, pluralize } from '@/lib/util'
 import { ReviewStateIconLink } from '@/subject/ReviewStateMarker'
+import { FormLabel } from '@/common/forms'
+import { ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/24/solid'
+import { BlobListLightbox } from '@/common/BlobListLightbox'
+import { PropsOf } from '@/lib/types'
 
 export function BlobList(props: {
   name: string
   disabled?: boolean
+  authorDid: string
   blobs: ToolsOzoneModerationDefs.BlobView[]
 }) {
   const { name, disabled, blobs } = props
+  const [lightboxImageIndex, setLightboxImageIndex] = useState(-1)
+
   return (
     <fieldset className="space-y-5 min-w-0">
-      {!blobs.length && <div className="text-sm text-gray-400">None found</div>}
-      {blobs.map((blob) => {
+      {!blobs.length ? (
+        <div className="text-sm text-gray-400">No blobs found</div>
+      ) : (
+        <BlobListLightbox
+          blobs={blobs}
+          authorDid={props.authorDid}
+          slideIndex={lightboxImageIndex}
+          onClose={() => setLightboxImageIndex(-1)}
+        />
+      )}
```

Comment:
Nit: for the sake of readability I would invert the conditions `blobs.length > 0 ? <BlobListLightbox /> : <div />`